src/Microsoft.Azure.WebJobs.Host.Rpc/WebJobs.Host.Rpc.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <PackageId>Microsoft.Azure.WebJobs.Host.Rpc</PackageId>
+    <AssemblyName>Microsoft.Azure.WebJobs.Host.Rpc</AssemblyName>
+    <RootNamespace>Microsoft.Azure.WebJobs.Host.Rpc</RootNamespace>
+    <Description>This package contains the runtime assemblies for Microsoft.Azure.WebJobs.Host.Rpc. For more information, please visit http://go.microsoft.com/fwlink/?LinkID=320971</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Azure.WebJobs.Host\WebJobs.Host.csproj" />
```

Comment:
This p2p reference is actually not needed/used - but figured I would set the dependency chain in place now.